src/general/GameWorld.cs

```diff
@@ -184,7 +184,7 @@ public Species CreateMutatedSpecies(Species species)
     /// </summary>
     public bool IsAutoEvoFinished(bool autostart = true)
     {
-        if (autoEvo == null && autostart)
+        if ((autoEvo == null || !autoEvo.Running) && autostart)
```

Comment:
Could you try to debug *how* an auto-evo run that doesn't get started is created?
Taking a quick look it looks like, the run is created for the external effect of player reproducing.
So the logic here is wrong. A new run should *not* be created if there is a run.
Instead there should be a check to start the run if autostart && !autoEvo.Running.